pyproject.toml

```diff
@@ -63,3 +63,4 @@ kubernetes = "^28.1.0b1"
 python-benedict = "^0.32.0"
 packaging = "^23.1"
 python-simple-logger = "^1.0.6"
+jinja2 = "^3.1.2"
```

Comment:
Why this is needed? Where it's used?

Reply:
https://github.com/RedHatQE/openshift-python-wrapper/blob/main/tests/utils.py#L3